<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XGenericTableHexWidget</class>
 <widget class="QWidget" name="XGenericTableHexWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="XGenericTableWidget" name="widgetMain" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidgetHex">
      <widget class="QWidget" name="pageEmpty"/>
      <widget class="XGenericHexWidget" name="pageHex"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XGenericTableWidget</class>
   <extends>QWidget</extends>
   <header>xgenerictablewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XGenericHexWidget</class>
   <extends>QWidget</extends>
   <header>xgenerichexwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
